mypy_django_plugin/transformers/managers.py

```diff
@@ -54,9 +177,11 @@ def create_new_manager_class_from_from_queryset_method(ctx: DynamicClassDefConte
     new_manager_info.defn.type_vars = base_manager_info.defn.type_vars
     new_manager_info.defn.line = ctx.call.line
     new_manager_info.metaclass_type = new_manager_info.calculate_metaclass_type()
-
-    current_module = semanal_api.cur_mod_node
-    current_module.names[ctx.name] = SymbolTableNode(GDEF, new_manager_info, plugin_generated=True)
+    # Stash the queryset fullname which was passed to .from_queryset
+    # So that our 'resolve_manager_method' attribute hook can fetch the method from that QuerySet class
+    new_manager_info.metadata.setdefault("django", {})
+    new_manager_info.metadata["django"].setdefault("from_queryset_manager", {})
+    new_manager_info.metadata["django"]["from_queryset_manager"] = derived_queryset_fullname
```

Comment:
No need for `.setdefault` here. `TypeInfo`(`new_manager_info`) is newly created which means that `metadata` is an empty dict.

```suggestion
    new_manager_info.metadata["django"] = {"from_queryset_manager": derived_queryset_fullname}
```